### DISCUSSION TITLE
After $subscribe is packaged, the events entered into the mutation will always be undefined, resulting in the invalidation of $subscribe

### DISCUSSION THREAD

**Discussion by KILROYLYK** [vuejs/pinia#1654] [Help and Questions] [answered]
### Reproduction

无

### Steps to reproduce the bug

Sorry for my poor English

Technology stack: TS + webpack + vue3 + Pinia

When packaging to the formal environment, it was found that the subscribe was not executed
Print the mutation of the subscription at the same location, and find:
When developing the environment, mutation is an object or array with events
But after packaging, the events of mutation are always undefined
Because the key in events is required to determine whether the event needs to be executed, it is invalid

### Expected behavior

<img width="384" alt="image" src="https://user-images.githubusercontent.com/22953292/189894523-7be9329e-a8ce-4e8f-a838-901ec20d52d2.png">

### Actual behavior

<img width="568" alt="image" src="https://user-images.githubusercontent.com/22953292/189894612-dc1aeb87-fa17-4b3d-a36c-cd78dca2c847.png">

### Additional information

A small suggestion is that events can be fixed as an array, otherwise you need to judge the type and then get the key

**Comment by posva** [comment, marked as answer]
This is expected [and documented](https://pinia.vuejs.org/api/interfaces/pinia.SubscriptionCallbackMutationPatchFunction.html#events). It's important to use `events` only for [debugging purposes](https://vuejs.org/guide/extras/reactivity-in-depth.html#component-debugging-hooks)

**Reply by KILROYLYK** [reply]
Thank you. I was careless
